### PR TITLE
Add blue stuff for reuse

### PR DESCRIPTION
**Pull request by TheDoubleK**
New blue styles required for this story: https://www.pivotaltracker.com/story/show/138703089

# 1. Adds a blue banner style
screen shot 2017-02-17 at 17 27 47

# 2. Extends the validation messages pattern with a blue version for all question types
screen shot 2017-02-17 at 17 23 14

screen shot 2017-02-17 at 17 23 01

screen shot 2017-02-17 at 17 22 46

screen shot 2017-02-17 at 17 22 31

screen shot 2017-02-17 at 17 22 15

### CODE REVIEW

File: toolkit/templates/forms/pricing.html [hunk 1]
```diff
@@ -30,6 +32,10 @@
     <span class="validation-message">
       {{ error }}
     </span>
+    {% elif message %}
+    <span class="message-message">
```
Comment (tombye) [inline toolkit/templates/forms/pricing.html]: How about `message-text` or maybe `message-content` instead?

Comment (TheDoubleK) [inline toolkit/templates/forms/pricing.html]: I was trying to follow the pattern of `validation-wrapper` and `validation-message`.  But yeah, I knew at the time this is weird.

Comment (TheDoubleK) [inline toolkit/templates/forms/pricing.html]: There is also prior art... cf. `temporary-message-message` in `_temporary_message.scss`

**Comment by TheDoubleK** [comment]
Changed the class name.  Gonna take the previous :+1:

**Pull request merged**